presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -87,6 +89,7 @@
     private final ResourceGroupId id;
     private final BiConsumer<InternalResourceGroup, Boolean> jmxExportListener;
     private final Executor executor;
+    private final boolean isStatic;
```

Comment:
nit: `s/isStatic/static/`. By convention boolean variables in java are declared without `is`. Geter names for boolean variables are constructed as `is` + `VariableName`.